addons/binding/org.openhab.binding.elkm1/src/main/java/org/openhab/binding/elkm1/internal/elk/message/ZoneDefinition.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
```

Comment:
It seems that this file has a missing, outdated or otherwise non standard copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.